### ISSUE TITLE
Convert image to vhdx for use with Hyper-V

### ISSUE THREAD

**Issue by gbraad** [code-ready/snc#51]
Needs additional packages and converted to a dynamic virtual disk:

```
$ ${SSH} core@api.${CRC_VM_NAME}.${BASE_DOMAIN} rpm-ostree install hyperv-daemons
$ qemu-img convert -f qcow2 -O vhdx -o subformat=dynamic crc.qcow2 crc.vhdx
```

**Comment by gbraad** [comment]
might have run into issues with https://github.com/code-ready/snc/blob/dd3c747912d5aae08ab295749cb9e1f4e1d342e8/createdisk.sh#L52

**Comment by gbraad** [comment]
Needs hyper-daemons to be installed inside the image:

```
$ rpm-ostree install hyperv-daemons
````

**Comment by praveenkumar** [comment]
fixed by #64
